<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector converter</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QPushButton" name="inputPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Input file...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="inputL">
     <property name="text">
      <string>&lt;input file&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="functionCB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="runPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Run...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="outputL">
     <property name="text">
      <string>&lt;output file&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="outputPB">
     <property name="text">
      <string>Output file...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
